src/Routes.js

```diff
@@ -1,17 +1,42 @@
 import { Route, Switch, Redirect } from 'react-router-dom';
 import PropTypes from 'prop-types';
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 
 import asyncComponent from './Utilities/asyncComponent';
+import axios from 'axios';
+import AsynComponent from '@redhat-cloud-services/frontend-components/AsyncComponent';
+import { ErrorState } from '@redhat-cloud-services/frontend-components';
 
 const DriftPage = asyncComponent(() => import ('./SmartComponents/DriftPage/DriftPage'));
 const BaselinesPage = asyncComponent(() => import ('./SmartComponents/BaselinesPage/BaselinesPage'));
 const EditBaselinePage = asyncComponent(() => import ('./SmartComponents/BaselinesPage/EditBaselinePage/EditBaselinePage'));
 
 const InsightsRoute = ({ component: Component, title, ...rest }) => {
     title ? document.title = title : null;
+    const INVENTORY_TOTAL_FETCH_URL = '/api/inventory/v1/hosts';
+    const [ hasSystems, setHasSystems ] = useState(true);
```

Comment:
I'd suggest to set it to false by default. This way, we can make sure we don't show the tables and content in the loading state:
```suggestion
    const [ hasSystems, setHasSystems ] = useState(false);
```